src/analysis/multiSpecies/SteadyCom/subroutines/setCplexParam.m

```diff
@@ -0,0 +1,114 @@
+function LP = setCplexParam(LP, solverParams,verbFlag)
```

Comment:
Thanks @laurentheirendt. I have removed the lines you mentioned. And I have moved `setCplexParam` and `checkSolFeas` to `src/base/solvers`. Other files in `subroutines` are very specific to the SteadyCom functions. 
In the section for `ibm_cplex` in `solveCobraLP`, assigning all parameters is actually in the to-do list (line 1205). The `setCplexParam` can have this nicely done because the Cplex parameter contains structures within structures within structures and needs quite many lines to do it.
`checkSolFeas` returns the infeasibility of solutions given the problem. Is there a function doing this in the toolbox? It is needed because solvers can sometimes return very specific solution status and there may be new status numbers in newer versions so the solution.stat may not always be the best indicator for feasibility.